README.md

```diff
@@ -27,7 +27,7 @@ It’s a React.js-based browser application with a Node.js backend with the foll
     yarn install
     ```
 
-    *Note: you might need Python 2 and the g++ compiler in order to build & install `node-zopfli`, one of our dependencies. If you are also running [`web-monitoring-processing`](https://github.com/edgi-govdata-archiving/web-monitoring-processing) (which uses Python 3), tools like [Conda](https://conda.io/) or [Pyenv](https://github.com/pyenv/pyenv) can help you manage multiple versions of Python.*
+    *Note: you might need Python 2 and the g++ compiler (on OS X, Xcode command line tools includes g++; on Debian systems [like Ubuntu], it’s included in the `build-essential` package) in order to build & install `node-zopfli`, one of our dependencies. If you are also running [`web-monitoring-processing`](https://github.com/edgi-govdata-archiving/web-monitoring-processing) (which uses Python 3), tools like [Conda](https://conda.io/) or [Pyenv](https://github.com/pyenv/pyenv) can help you manage multiple versions of Python.*
```

Comment:
On yum-/dnf-based systems like CentOS, RHEL, and Fedora you can install the standalone gcc-c++ package, or install it as part of the "Development Tools" package group.

Reply:
Thanks for the pointer on this; I’ve updated the text!